### PR TITLE
Feature/r0.9.9 doc changes

### PR DESCRIPTION
**Pull request by awholegunch**
A small change with a great impact: adds highlighting to parsed-literal code blocks.

Note that in many cases, you don't need to add backslashes for the parsed-literals; just if you are using underscores.

This will be very useful in CDAP, as it will allow simplification of building the docs.

See the [installation page for the example output](http://stg-web101.sw.joyent.continuuity.net/coopr/0.9.9-r0.9.9_doc_changes/en/installation/packages.html#yum).

(I know. It doesn't look impressive. Looks like all the other code blocks. But that's the point...)

**Edit:** Added new screen-snapshots at:
- [Admin](http://stg-web101.sw.joyent.continuuity.net/coopr/0.9.9-r0.9.9_doc_changes/en/guide/admin/ui.html)
- [User guide](http://stg-web101.sw.joyent.continuuity.net/coopr/0.9.9-r0.9.9_doc_changes/en/guide/user/index.html)
- [Superadmin](http://stg-web101.sw.joyent.continuuity.net/coopr/0.9.9-r0.9.9_doc_changes/en/guide/superadmin/tenants.html)
- [Quickstart](http://stg-web101.sw.joyent.continuuity.net/coopr/0.9.9-r0.9.9_doc_changes/en/guide/quickstart/index.html)

There is still text editing to be completed.

### CODE REVIEW

**Comment by wolf31o2** [comment]
:+1: LGTM... should this be applied to other parts of the docs?

**Comment by awholegunch** [comment]
It could be applied elsewhere. The _replace_ could be added to the `conf.py`, and then it would be available on every page automatically.

File: coopr-docs/docs/source/installation/packages.rst [hunk 1]
```diff
@@ -138,20 +138,25 @@ Yum
...
 
-.. parsed-literal::
-  $ sudo yum localinstall coopr-server-\ |version|\ .el6.x86_64.rpm
-  $ sudo yum localinstall coopr-provisioner-\ |version|\ .el6.x86_64.rpm
-  $ sudo yum localinstall coopr-ui-\ |version|\ .el6.x86_64.rpm
+.. container:: highlight
 
+  .. parsed-literal::
+   
+    |$| sudo yum localinstall coopr-server-|version|.el6.x86_64.rpm
```
Comment (duddun) [inline coopr-docs/docs/source/installation/packages.rst:145]: How is |$| supposed to render?

Comment (awholegunch) [inline coopr-docs/docs/source/installation/packages.rst:145]: "|$|" renders as a "$", but highlighted in orange. See the staged example. It's hard to see the difference, because now it matches all the other examples.

**Comment by awholegunch** [comment]
**Note:** text editing to match new screen captures to be completed. Don't merge yet!...

**Comment by awholegunch** [comment]
I've update the text and images; and updated [the staged version](http://stg-web101.sw.joyent.continuuity.net/coopr/0.9.9-r0.9.9_doc_changes/en/index.html); review away!

In particular, look at the links listed in the first comment of this PR.

File: coopr-docs/docs/source/guide/admin/ui.rst [hunk 2]
```diff
@@ -435,63 +456,79 @@ cannot be placed onto a cluster without also placing the services it requires. T
...
-.. figure:: /_images/admin/ui/service_create_dependencies.png
+.. figure:: /_images/admin/ui/ui_service_create_dependencies.png
     :align: center
     :width: 800px
     :alt: Service Dependencies
     :figclass: align-center
 
+Provides
+--------
+The ``Provides`` field define an extra level of indirection when specifying dependencies.
```
Comment (dereklwood) [inline coopr-docs/docs/source/guide/admin/ui.rst]: defines ?

File: coopr-docs/docs/source/guide/quickstart/index.rst [hunk 3]
```diff
@@ -408,15 +428,18 @@ Once creation is complete, the cluster is ready for use.
 
 For more information on your cluster, click on the name 'hadoop-quickstart' on the
 Clusters' home screen. On this cluster description screen, nodes are grouped together by the set
-of services that are available on them. To see node details, click on the white triangles next to each
-service set to expand the list. The expanded list shows a list of attributes for each node.
+of services that are available on them. To see node details, click on the descriptions
+next to the number of nodes ('2 noes with 5 services each') to expand the list. The
```
Comment (dereklwood) [inline coopr-docs/docs/source/guide/quickstart/index.rst]: nodes

File: coopr-docs/docs/source/guide/quickstart/index.rst [hunk 4]
```diff
@@ -408,15 +428,18 @@ Once creation is complete, the cluster is ready for use.
...
     :align: center
     :width: 800px
     :alt: Cluster description and details
     :figclass: align-center
 
-In this example, there is 1 master node that contains the 'hbase-master', 'hadoop-hdfs-namenode', 'zookeeper-server', and
-'hadoop-yarn-resourcemenager' services. There are also 4 slave nodes that contain the 'hbase-regionserver', 'hadoop-yarn-nodemanager',
-and 'hadoop-hdfs-datanode' services.
+In this example, there is 1 master node that contains the 'base',
+'continuuity-sensu-monitoring', 'hadoop-hdfs-namenode', 'hadoop-yarn-resourcemanager',
```
Comment (dereklwood) [inline coopr-docs/docs/source/guide/quickstart/index.rst]: continuuity-sensu-monitoring is one of our internal services.  it's probably ok for now since this is just an illustrative example, but it would be good to remove in the future

Comment (wolf31o2) [inline coopr-docs/docs/source/guide/quickstart/index.rst]: We shouldn't document internal services, IMO. We, especially, should not introduce new documentation with them.

File: coopr-provisioner [hunk 5]
```diff
@@ -1 +1 @@
-Subproject commit 8cae541629526e370c0482670cd15c4ad64c6a77
+Subproject commit 5328a380f7ab0c362ad99e9c151a859b1f350d3e
```
Comment (dereklwood) [inline coopr-provisioner]: why is this here?  we need to doublecheck this

Comment (dereklwood) [inline coopr-provisioner]: confirmed this is bad... pls revert and leave it at 8cae541629526e370c0482670cd15c4ad64c6a77

File: coopr-docs/docs/source/guide/quickstart/index.rst (hunk 4 above)
Comment (awholegunch) [inline coopr-docs/docs/source/guide/quickstart/index.rst]: Removed the two references.

File: coopr-docs/docs/source/guide/quickstart/index.rst (hunk 3 above)
Comment (awholegunch) [inline coopr-docs/docs/source/guide/quickstart/index.rst]: Fixed.

File: coopr-docs/docs/source/guide/admin/ui.rst (hunk 2 above)
Comment (awholegunch) [inline coopr-docs/docs/source/guide/admin/ui.rst]: Fixed.

File: coopr-provisioner (hunk 5 above)
Comment (awholegunch) [inline coopr-provisioner]: will do...

**Comment by wolf31o2** [comment]
:+1: LGTM
